src/Lifi.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable max-len */
```

Comment:
Eslint allows to define different rules for different types of lines. We should rather just increase/disable the max-length for comments instead of disabling it completely.

https://eslint.org/docs/rules/max-len